config/src/main/resources/calaveras.json

```diff
@@ -1,22 +1,24 @@
 {
   "config": {
-    "chainId": 1642,
+    "chainId": 123,
     "homesteadBlock": 0,
-    "daoForkBlock": 0,
     "eip150Block": 0,
+    "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "eip155Block": 0,
     "eip158Block": 0,
     "byzantiumBlock": 0,
+    "constantinopleBlock": 0,
     "petersburgBlock": 0,
     "istanbulBlock": 0,
     "berlinBlock": 0,
     "londonBlock": 500,
     "clique": {
-      "blockperiodseconds": 30,
-      "epochlength": 30000
+      "period": 30,
+      "epoch": 30000
```

Comment:
shouldn't these be epochlengh and blockperiodseconds ?

Reply:
Yes I updated just before your comment 😄